### DISCUSSION TITLE
Debug using IIS express

### DISCUSSION THREAD

**Discussion by kwaazaar** [CZEMacLeod/MSBuild.SDK.SystemWeb#33] [Q&A] [answered]
Is VS supposed to see the project as a regular web project? In my case that doesn't work. For debugging, I must manually configure IISExpress.exe to be executed.

Maybe that's related to getting this error when trying to install the package?:
`Package 'MSBuild.SDK.SystemWeb 4.0.72' has a package type 'MSBuildSdk' that is not supported by project 'Middelware.Cloud.Wcf'.`

**Comment by kwaazaar** [comment, marked as answer]
So to answer this myself: IISExpress does work, but I had to remove my manually created launchSettings.json first.

I read somewhere I should use this SDK project, but the readme doesn't really describe what it does/why I need it.
Only when I found [this issue](https://github.com/dotnet/project-system/issues/2670), I understood that it combines all manual changes to the csproj required into a single SDK-reference. Pretty neat :-)

**Reply by CZEMacLeod** [reply]
I think you have worked it out. This is an SDK project, not an installable nuget package.
The readme does include a section called How can I use these SDKs? which does cover setting the project SDK.
The templates also include details on how to install them and create a new project of this type which you could then use as a reference.
If you think that a getting started guide of some sort would be helpful, then I'd be very happy to accept a PR with the additional documentation.
